jacodb-analysis/build.gradle.kts

```diff
@@ -6,20 +6,26 @@ plugins {
     kotlin("plugin.serialization") version "1.7.20"
 }
 
+repositories {
+    maven(url = "https://jitpack.io")
+}
+
 dependencies {
     api(project(":jacodb-core"))
     api(project(":jacodb-api"))
 
     testImplementation(testFixtures(project(":jacodb-core")))
     testImplementation(project(":jacodb-api"))
-    testImplementation(group = "javax.servlet", name = "servlet-api", version = "2.5")
     testImplementation(group = "org.junit.jupiter", name = "junit-jupiter-params", version = "5.9.2")
-    testImplementation(files("src/test/resources/juliet.jar"))
     testImplementation(files("src/test/resources/pointerbench.jar"))
     testImplementation(group = "joda-time", name = "joda-time", version = "2.12.5")
+    testImplementation("com.github.Lipen.juliet-test-suite:support:1.3.6")
```

Comment:
The ZIP archive from NIST does not contain the LICENSE file, either. Should I simply create one (CC0)?

Regarding the repo: I can re-upload the project under UTBot org, without "forking" (I bet I cannot simply "Transfer repo", since it would probably remain to be "forked"), but what about the git history? The initial commits were made by find-sec-bugs's members.

Reply:
I've moved Juliet repo to https://github.com/UnitTestBot/juliet-java-test-suite